app/src/main/java/com/powerhungers/fomezero/feature/producer/basket/domain/model/Basket.kt

```diff
@@ -0,0 +1,11 @@
+package com.powerhungers.fomezero.feature.producer.basket.domain.model
+
+enum class BasketType() {
```

Comment:
```suggestion
enum class BasketType {
```
como essa classe nao tera um construtor nao tem necessidade de parenteses